src/tests/work-item/work-item-list/assigne.spec.js

```diff
@@ -0,0 +1,149 @@
+      page.clickWorkItemQuickAdd();
+      page.typeQuickAddWorkItemTitle(workItemTitle);
+      page.clickQuickAddSave().then(function() {
+      page.workItemViewId(page.workItemByTitle(workItemTitle)).getText().then(function (text) {
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      expect(detailPage.workItemDetailAssigneeIcon().isPresent()).toBeTruthy();
+      expect(detailPage.workItemDetailAssignee().isPresent()).toBeTruthy();
+      detailPage.clickWorkItemDetailCloseButton();
+      });
+    });
+  }); 
+ 
+  /**Test to update the assigned user */ 
+  it('Test to update the assigned user  -phone ', function() {
+      page.workItemViewId(page.firstWorkItem).getText().then(function (text) { 
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      detailPage.workItemDetailAssigneeIcon().click();
+      detailPage.setWorkItemDetailAssigneeSearch("Walter Mitty",false);
+      detailPage.clickAssignedUserDropDownList("Walter Mitty");
+      expect(detailPage.details_assigned_user().getText()).toContain("Walter Mitty");
+      detailPage.clickWorkItemDetailCloseButton();
+     });
+  }); 
+   
+ /**User can read , update , remove assignee  */
+   it('User can read , update , remove assignee  -phone ', function() {
+      var workItemTitle = "The test workitem title";
+      var workItemUpdatedTitle = "The test workitem title - UPDATED";
+      page.clickWorkItemQuickAdd();
+      page.typeQuickAddWorkItemTitle(workItemTitle);
+      page.clickQuickAddSave().then(function() {
+      page.workItemViewId(page.workItemByTitle(workItemTitle)).getText().then(function (text) {
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      detailPage.workItemDetailAssigneeIcon().click();
+      detailPage.setWorkItemDetailAssigneeSearch("Harry Potter",false);
+      detailPage.clickAssignedUserDropDownList("Harry Potter");
+      expect(detailPage.details_assigned_user().getText()).toContain("Harry Potter");
+      detailPage.details_assigned_user().click();
+      detailPage.clickworkItemDetailUnassignButton();
+      });
+    });
+  }); 
+ /**User can Cancel assignee  */
+   it('User can Cancel assignee -phone ', function() {
+      var workItemTitle = "The test workitem title";
+      var workItemUpdatedTitle = "The test workitem title - UPDATED";
+      page.clickWorkItemQuickAdd();
+      page.typeQuickAddWorkItemTitle(workItemTitle);
+      page.clickQuickAddSave().then(function() {
+      page.workItemViewId(page.workItemByTitle(workItemTitle)).getText().then(function (text) {
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      detailPage.workItemDetailAssigneeIcon().click();
+      detailPage.setWorkItemDetailAssigneeSearch("Harry Potter",false);
+      detailPage.clickAssignedUserDropDownList("Harry Potter");
+      expect(detailPage.details_assigned_user().getText()).toContain("Harry Potter");
```

Comment:
I mentioned this earlier, but, I think that it would be a good standard practice to adopt for tests that modify properties in the detail page, close and then re-open the detail page before attempting to verify the modifications.